docs/querying/sql-aggregations.md

```diff
@@ -139,6 +139,16 @@ Load the [DataSketches extension](../development/extensions-core/datasketches-ex
 |`DS_QUANTILES_SKETCH(expr, [k])`|Creates a [Quantiles sketch](../development/extensions-core/datasketches-quantiles.md) on the values of `expr`, which can be a regular column or a column containing quantiles sketches. The `k` parameter is described in the Quantiles sketch documentation.<br/><br/>See the [known issue](sql-translation.md#approximations) with this function.|`'0'` (STRING)|
 
 
+### Tuple sketch functions
+
+Load the [DataSketches extension](../development/extensions-core/datasketches-extension.md) to use the following functions.
```

Comment:
I believe all the sketches in the `druid-datasketches` module are currently described in more detail at the linked page: https://github.com/apache/druid/blob/master/docs/development/extensions-core/datasketches-extension.md

That page also already links to the [Apache DataSketches](https://datasketches.apache.org/) project page which has its own extensive documentation. Is that not sufficient?

Reply:
IMO we don't need this description added here, since documentation already exists for tuple sketches generally: https://druid.apache.org/docs/latest/development/extensions-core/datasketches-tuple.html. Admittedly it could use improvement— they are quite terse when it comes to why you might want to use the sketches in the first place— but I'd consider that out of scope for a patch that is about to add SQL bindings.